src/libraries/GetTradeableOrder.sol

```diff
@@ -2,7 +2,7 @@
 pragma solidity ^0.8.24;
 
 import {Math} from "lib/openzeppelin/contracts/utils/math/Math.sol";
-import {IConditionalOrder, GPv2Order, IERC20} from "lib/composable-cow/src/BaseConditionalOrder.sol";
+import {GPv2Order, IERC20} from "cowprotocol/contracts/libraries/GPv2Order.sol";
```

Comment:
This is a bit weird because `cowprotocol` is a transitive dependency and somewhat comes out of nowhere (it works because Foundry automatically injects remappings for all transitive dependencies and `composable-cow` needs it). The syntax we'd use in this case would try to make it clearer where the dependency comes from:

```suggestion
import {GPv2Order, IERC20} from "lib/composable-cow/lib/cowprotocol/src/contracts/libraries/GPv2Order.sol";
```

More in general, it would be good for the entire repo to eliminate `composable-cow` and use `cowprotocol` as a direct dependency, but this is too much work for deprecated code!

Anyway, if you find your current version better than my suggestion I'm ok with keeping it like that.
In general, the test interface is only used by (1) our back-end for testing and (2) as guidance for the Balancer AMM, so we'd only change this for these two use cases.

Reply:
i tried to follow the import i've seen on the other contracts as a standard, they were all importing from `cowprotocol`

although, i find that common names could be useful for imports, since the version `lib/composable-cow/lib/cowprotocol/src` would need to be inside our remappings, and in this way, we can set the remappings to the `cowprotocol` we also inherit locally, and should read the file exactly from where we want

Reply:
Oh, I see this is how it's done in the test interface. Yeah, I see how it makes further remappings useful, for consistency we should merge it as it is.